editor.planx.uk/src/pages/FlowEditor/components/Flow/FeedbackPage.tsx

```diff
@@ -46,20 +90,216 @@ const getDetailedFeedback = async (feedbackId: number) => {
     query: GET_FEEDBACK_BY_ID_QUERY,
     variables: { feedbackId },
   });
-  console.log(detailedFeedback);
+
+  const combinedHelpText = `${detailedFeedback.helpText || ""} ${
+    detailedFeedback.helpDefinition || ""
+  } ${detailedFeedback.helpSources || ""}`.trim();
+  const truncatedHelpText =
+    combinedHelpText.length > 65
+      ? `${combinedHelpText.slice(0, 65)}...`
+      : combinedHelpText;
```

Comment:
:+1: clever! I always forget the css syntax to do this, good reminder it can be clearly handled this way too